core/controllers/editor.py

```diff
@@ -468,21 +453,6 @@ def post(self, unused_exploration_id):
         })
 
 
-class ExplorationResourcesHandler(EditorHandler):
```

Comment:
I don't think this handler is used in the front-end. Can anyone confirm this?

Reply:
I agree, we can drop it and drop the corresponding stanza in main.py. Good catch!